lib/ansible/modules/web_infrastructure/letsencrypt.py

```diff
@@ -96,13 +96,20 @@
       - "The value that must be used here will be provided by a previous use
          of this module."
   dest:
-    description: The destination file for the certificate.
+    description:
+      - "The destination file for the certificate."
+      - "If C(fullchain_dest) is specified, C(dest) can be left away."
     required: true
     aliases: ['cert']
-  fullchain:
-    description: Include the full certificate chain in the destination file.
+  fullchain_dest:
+    description:
+      - "The destination file for the full chain (i.e. certificate followed
+         by chain of intermediate certificates)."
+      - "If C(dest) is specified, C(fullchain_dest) can be left away."
+    required: true
```

Comment:
according code, it is not required.

Reply:
See below. How should this (i.e. at least one of `dest` and `fullchain` is specified) be marked here? My idea was to mark both as required, but add some text that only one needs to be specified. Should I mark both as not required, and add a note to each that at least one of them must be specified?

Reply:
yes, both not required, but put a note in the doc that "Either C(dest) or C(fullchain_dest) is required."

Reply:
I've updated the documentation. I hope it's better explained now.